src/main/lrsql/system/webserver.clj

```diff
@@ -47,12 +52,13 @@
                                          [i/error-interceptor
                                           (handle-json-parse-exn)]
                                          oidc-interceptors)})
-             (add-admin-routes {:lrs               lrs
-                                :exp               jwt-exp
-                                :leeway            jwt-lwy
-                                :secret            private-key
-                                :enable-admin-ui   enable-admin-ui
-                                :oidc-interceptors oidc-admin-interceptors}))]
+             (add-admin-routes {:lrs                  lrs
+                                :exp                  jwt-exp
+                                :leeway               jwt-lwy
+                                :secret               private-key
+                                :enable-admin-ui      enable-admin-ui
+                                :oidc-interceptors    oidc-admin-interceptors
+                                :oidc-ui-interceptors oidc-admin-ui-interceptors}))]
```

Comment:
I probably should've mentioned this earlier in the OIDC dev pipeline, but I think it might be a good idea to refactor all of the interceptor/route building to be in the `admin.routes` namespace, with only config args being passed in here (similar to how you pass in `enable-admin-ui` here while the actual admin UI interceptors are added in the other ns).

Reply:
I was thinking of moving the definitions of `oidc-interceptors`, `oidc-admin-interceptors`, and `oidc-admin-ui-interceptors` to the `routes` namespace. But after looking at the code again, I realized that it wouldn't make sense to put _all_ the interceptors in that namespace since not all of them are admin-related. Since there's only two sets of admin interceptors being passed here, I now think it's fine as-is.

Reply:
Cool, makes sense. I _did_ notice in the process that these lines: https://github.com/yetanalytics/lrsql/blob/SQL-138/src/main/lrsql/system/webserver.clj#L35-L44 can actually get pushed up into OIDC init because they are solely concerned with accessing functions from there, so I am gonna do a little refactor there to keep the webserver file a bit more readable.

Reply:
https://github.com/yetanalytics/lrsql/pull/210/commits/ee7d49077c053ed9e998b8c9988683f0158c0b34